packages/react-scripts/template/README.md

```diff
@@ -1009,6 +1010,12 @@ Learn more about React Storybook:
 
 You can turn your React app into a [Progressive Web App](https://developers.google.com/web/progressive-web-apps/) by following the steps in [this repository](https://github.com/jeffposnick/create-react-pwa).
 
+## Git Repository
```

Comment:
@gaearon Not quite sure what you want me to do. Do you want me to leave it as it is there already or delete it?